packages/themes/ecl/src/ecl-ec/components/table-stateful.scss

```diff
@@ -19,6 +18,11 @@
 		&__pagination {
 			overflow-x: auto;
 			overflow-y: hidden;
+			&__pagination--bottom {
```

Comment:
Hier hat sich noch ein kleiner Fehler eingeschlichen.
Wenn der media query nur auf dem wrapper notwendig ist, diesen Teil bitte ganz raus nehmen

Reply:
ist in der tat unnötig